app/models/product.rb

```diff
@@ -1,5 +1,8 @@
 class Product < ApplicationRecord
   belongs_to :user
   has_many :rentals
-end
 
+  validates :description, presence: true
+  validates :price, presence: true
```

Comment:
price validation must be positive, so this validation should be implemented here too